zdm/iteration.py

```diff
@@ -737,7 +892,89 @@ def calc_likelihoods_2D(grid,survey,doplot=False,norm=True,psnr=True,printit=Fal
         plt.savefig('1d_dm_fit.pdf')
         plt.close()
     
-    ###### Calculates p(E | z,DM) ########
+    ################ Calculates p(tau,w| total width) ###############
+    if ptauw:
```

Comment:
This looks pretty duplicated with the 1D code.  Can one avoid the repetition with a method?